src/test/fuzz/package_eval.cpp

```diff
@@ -145,7 +145,7 @@ FUZZ_TARGET(tx_package_eval, .init = initialize_tx_pool)
     }
 
     auto outpoints_updater = std::make_shared<OutpointsUpdater>(mempool_outpoints);
-    RegisterSharedValidationInterface(outpoints_updater);
+    g_setup->m_node.validation_signals->RegisterSharedValidationInterface(outpoints_updater);
```

Comment:
nit:
```suggestion
    node.validation_signals->RegisterSharedValidationInterface(outpoints_updater);
```

(same for the ones below)